.github/workflows/sync-rest.yml

```diff
@@ -253,6 +277,7 @@ jobs:
         env:
           TARGET_REPO_FULLNAME: ${{ matrix.repo }}
           BRANCH_NAME: ${{ steps.branch.outputs.name }}
+          TRACKING_ISSUE_URL: ${{ needs.open-tracking-issue.outputs.issue-url }}
```

Comment:
Does the `needs.` prefix mean that this will fail it the issue could not be created?

Reply:
If the issue could not be created, the workflow won't even reach this stage.

The `needs.` prefix here is used to access the output of the needed job  `open-tracking-issue`. It refers back to L30, which refers to the output of the github-script step that creates the issue.